### ISSUE TITLE
Can you tag a new release for 5.7?

### ISSUE THREAD

**Issue by bbashy** [albertcht/invisible-recaptcha#93]
Thanks for the great work.

**Comment by albertcht** [comment]
Thanks for your reminding, I've released `v1.8.4`.

**Comment by jastend** [comment]
Just wanted to say thank you! I was wondering why I couldn't update.

**Comment by albertcht** [comment]
Hi @jastend ,

What do you mean `couldn't update`? Is there any error message?

**Comment by bbashy** [comment]
@albertcht Think they meant, they wondered why they couldn't update before this new release.

**Comment by jastend** [comment]
Correct. It was just a thank you for the quick update. Everything works like a charm now.

**Comment by albertcht** [comment]
Ok, I got it, no problem :)
